src/main/java/com/laserfiche/repository/api/clients/impl/AttributesClientImpl.java

```diff
@@ -36,21 +36,23 @@ public CompletableFuture<Attribute> getTrusteeAttributeValueByKey(String repoId,
                             Object body = httpResponse.getBody();
                             String jsonString = new JSONObject(body).toString();
                             return objectMapper.readValue(jsonString, Attribute.class);
-                        } catch (JsonProcessingException e) {
+                        } catch (JsonProcessingException | IllegalStateException e) {
                             e.printStackTrace();
                             return null;
                         }
                     } else {
                         Object body = httpResponse.getBody();
                         ProblemDetails problemDetails;
+                        Map<String, String> headersMap = getHeadersMap(httpResponse.getHeaders());
                         try {
                             String jsonString = new JSONObject(body).toString();
                             problemDetails = objectMapper.readValue(jsonString, ProblemDetails.class);
-                        } catch (JsonProcessingException e) {
-                            e.printStackTrace();
-                            return null;
+                        } catch (JsonProcessingException | IllegalStateException e) {
+                            throw new ApiException(httpResponse.getStatusText(), httpResponse.getStatus(), httpResponse
+                                    .getParsingError()
```

Comment:
I guess we are okay with `NullPointerException` for `getParsingError()`, `get()`, and `getOriginalBody()`?

Reply:
Good catch. Thanks. I fixed the issue.